config/webpacker.yml

```diff
@@ -83,7 +83,7 @@ production:
   <<: *default
 
   # Production depends on precompilation of packs prior to booting for performance.
-  compile: false
+  compile: true
```

Comment:
Why is this needed to be `true`?

Reply:
had some issues with webpacker and i was trying to fix it for a long time and one of the forums told me to change it and i forgot to change back.